core/themes/bartik/color/color.inc

```diff
@@ -16,7 +16,6 @@ $info = array(
     'text' => t('Text color'),
     'link' => t('Link color'),
     'menu' => 'Menu link',
-    'activemenu' => t('Active menu link color'),
```

Comment:
Was this removal intended?

Reply:
Yes. I forgot to mention it in my comment, but the theme doesn't actually use that setting, so I removed it.